gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenMathExpressionsSuite.scala

```diff
@@ -18,11 +18,40 @@ package org.apache.spark.sql.catalyst.expressions
 
 import org.apache.gluten.utils.BackendTestUtils
 
+import org.apache.spark.sql.GlutenQueryTestUtil.isNaNOrInf
 import org.apache.spark.sql.GlutenTestsTrait
 import org.apache.spark.sql.catalyst.dsl.expressions._
 import org.apache.spark.sql.types._
 
+import org.apache.commons.math3.util.Precision
+
+import java.nio.charset.StandardCharsets
+
 class GlutenMathExpressionsSuite extends MathExpressionsSuite with GlutenTestsTrait {
+  override protected def checkResult(
```

Comment:
Hi @taiyang-li, is this check also applied to Velox backend testing?

Reply:
Yes, I'll apply it only to CH backend.